Wire-Protocol.rst

```diff
@@ -54,45 +107,51 @@ The following table lists return codes. It is worth stressing that all codes fal
 * <0: not defined
 * 0: success
 * 1-99: warnings (request fulfilled but with some caveat)
-* >=100: error
-
-The errors are further subdivided, with each multiple of 100 naming a category and other values falling within that category.
-Any value not specified here may be defined within an individual mesh (client libraries are expected to be able to handle new retcode values, though obviously they may not know the description for them).
-Users are encouraged to define a new category if needed and to either leave a gap within the category or start from the upper end (from x99) to avoid conflicts with values which may be defined in future dripline versions.
+* 100-999: dripline error
+* >=1000: application errors
 
+Errors are subdivided into categories, with each multiple of 100 representing a category and values falling within that category.
+Dripline errors are covered by codes in the 100-999 range.
+Additional errors may be specified for a particular application of dripline.  These errors are covered by codes 1000 and above.
 
 ======= ===========
 Code    Description
 ======= ===========
-0       Success
-1       No action taken warning
-2-99    Unassigned, non-error warnings
-100     Generic AMQP Related Error
+0       **Success**
+1       **Generic Warning; No Action Taken**
+2-99    *Unassigned, Non-Error Warnings*
+100     **Generic AMQP Related Error**
 101     AMQP Connection Error
 102     AMQP Routing Key Error
-103-199 Unallocated AMQP Errors
-200     Generic Hardware Related Error
+103-199 *Unallocated AMQP Errors*
+200     **Generic Hardware Related Error**
 201     Hardware Connection Error
-202     Hardware no Response Error
-203-299 Unallocated Hardware Errors
-300     Generic Dripline Client Error
+202     Hardware No Response Error
+203-299 *Unallocated Hardware Errors*
+300     **Generic Dripline Client Error**
 301     No message encoding error
 302     Decoding Failed Error
-303     Payload related error
+303     Payload Related Error
 304     Value Error
 305     Timeout
-306     Method not supported
-307     Access denied
-308     Invalid key
+306     Method Not Supported
+307     Access Denied
+308     Invalid Key
 309     Deprecated Feature
-310-399 Unallocated Dripline errors
-400     Generic System Resource Error
-500     Generic DAQ Error
-501-998 Unallocated
-999     Unhandled core-language or dependency exceptions
+310-399 *Unallocated Dripline errors*
+400     **Generic Client Error**
```

Comment:
I don't think I understand the difference between the 300s and the 400s

Reply:
Hmm, that's a good point.  In the C++ the 300s are labeled as "message" errors, and I had assumed they were service-side errors.

Reply:
For the GitHub record, I'll review the C++ use of 300- and 400-class errors and see if there's a reasonable way to consolidate, or if we need two classes.

Reply:
I'm going to create a new issue to deal with sorting out return codes.  For now I'm going to resolve these questions and merge the branch.  Further modifications can be made in a new branch.